deps/flatgeobuf/geometryreader.h

```diff
@@ -0,0 +1,56 @@
+#ifndef FLATGEOBUF_GEOMETRYREADER_H
+#define FLATGEOBUF_GEOMETRYREADER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "lwgeom_log.h"
+#include "liblwgeom.h"
```

Comment:
this depends on lwgeom so definitely not third-party dependency.

Reply:
Banner added.